Source/JavaScriptCore/runtime/ModuleProgramExecutable.h

```diff
@@ -52,6 +52,8 @@ class ModuleProgramExecutable final : public GlobalExecutable {
         return bitwise_cast<ModuleProgramCodeBlock*>(Base::codeBlock());
     }
 
+    UnlinkedModuleProgramCodeBlock* getUnlinkedCodeBlock(JSGlobalObject*);
```

Comment:
nit: I would prefer that you add an enum class here like this:
```
enum class PossibleExceptionsExpected { Yes, No };
```

... and pass that into `getUnlinkedCodeBlock`.